docs/source/user/toc.rst

```diff
@@ -14,6 +14,7 @@ In the sidebar panel, you can number headings, collapse sections, and navigate i
 Here is an animation showing the table of content use with a notebook:
 
 .. image:: ./images/toc/toc.gif
+    :alt:
```

Comment:
A .gif file displaying how to use the table of contents feature within a notebook.

Reply:
Hey there, thanks for contributing! If you'd be able to edit your above comment into a [GitHub suggestion](https://hackmd.io/@marslee/BksnYfWVK), I'd appreciate so I can make sure you get credit in the commit history. If not, I'll still add it with an extended description that mentions you.

Reply:
```suggestion
    :alt: A .gif file displaying how to use the table of contents feature within a notebook.
```